tests/constraints.in

```diff
@@ -1,5 +1,5 @@
 # Known limitations for indirect/transitive dependencies.
 resolvelib < 1.1.0
-sphinx == 5.3.0
+sphinx == 6.2.1
```

Comment:
I would avoid bumping the Sphinx version in an 'unrelated' PR.

Reply:
sure. I just noticed that version was in use on devel and thought I'd correct it. I'll send a separate PR.